pkgs/by-name/me/memcachedTestHook/package.nix

```diff
@@ -0,0 +1,17 @@
+{
+  callPackage,
+  makeSetupHook,
+  memcached,
+  netcat,
+}:
+
+makeSetupHook {
+  name = "memcached-test-hook";
+  propagatedBuildInputs = [
+    memcached
+    netcat
+  ];
```

Comment:
I'm not entirely sure we want to propagate these packages. We could just use `buildInputs`, no? Or use `replaceVars` on the hook script.

Reply:
True, I pushed a fix using `substitutions` (`makeSetupHook` has no `buildInputs`)